nav2_behavior_tree/plugins/condition/goal_updated_condition.cpp

```diff
@@ -28,7 +28,7 @@ GoalUpdatedCondition::GoalUpdatedCondition(
 
 BT::NodeStatus GoalUpdatedCondition::tick()
 {
-  if (status() == BT::NodeStatus::IDLE) {
+  if (!BT::isStatusActive(status())) {
```

Comment:
I actually looked through your 3.5->4 migration guide and couldn't find a reference to it -- that's new to v4 that I think is worth calling out :-) 

I also went through the tutorial pages and didn't find it, but its hard to say if I just missed it since there isn't a search feature. The "Basic Concepts" mention success/failure/running but not skipped as well. This doesn't appear to be documented (?) 

That made it not possible for me to review the decorators, but otherwise, comments are for everything else

Reply:
Improved the documentation. Thanks for pointing this out, it was required.
https://www.behaviortree.dev/docs/migration